app/Models/FailedLog.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+namespace Rogue\Models;
+
+use Illuminate\Database\Eloquent\Model;
+
+class FailedLog extends Model
```

Comment:
Not necessary for this pull request, but I wonder if we should be handling this with [Laravel queues](https://laravel.com/docs/5.3/queues) since they have good support for [handling re-trying failed jobs](https://laravel.com/docs/5.3/queues#dealing-with-failed-jobs) built in.